switch/libsmb2/libsmb2-91d6c8a-fix_makefile_libnx_include.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/lib/Makefile.am b/lib/Makefile.am
+index 72a86ec..fe133eb 100644
+--- a/lib/Makefile.am
++++ b/lib/Makefile.am
+@@ -4,6 +4,7 @@ lib_LTLIBRARIES = libsmb2.la
+ 
+ libsmb2_la_CPPFLAGS = -I$(abs_top_srcdir)/include \
+ 		      -I$(abs_top_srcdir)/include/smb2 \
++			  -I$(DEVKITPRO)/libnx/include \
```

Comment:
Not sure why this is needed, but the only way I could get it to work.
Build doesn't seem to pick up the `libnx` includes from the `-isystem` already defined in `switchvars.sh`.

In `configure.ac` some of the header checks are failing.
e.g. `AC_CHECK_HEADERS([arpa/inet.h])` despite that header actually existing, it reports it as missing for some reason.